src/southbridge/amd/sr5650/bootblock.c

```diff
@@ -19,6 +47,7 @@ void sr5650_disable_pcie_bridge(void)
 	mask |= (1 << 20); /*GPP3b*/
 	reg = mask;
 	set_nbmisc_enable_bits(nb_dev, 0x0c, mask, reg);
+	enable_pcie_bar3();
```

Comment:
This function is public so maybe lets' move it to bootblock_early_southbridge_init in sb700/bootblock as other sr5650 into functions are there

Reply:
Done